client/js/screens/SingleSelfGuided/SingleSelfGuided.js

```diff
@@ -0,0 +1,168 @@
+                  />
+                </View>
+                <View style={styles.eventButton}>
+                  <Image
+                    source={require('../../assets/images/icFaveCircleDefault.png')}
+                  />
+                </View>
+              </View>
+            </View>
+          </View>
+
+          <View style={styles.eventScreenInfo}>
+            <View style={styles.eventDetails}>
+              <View style={styles.difficultyFilter}>
+                <Text style={styles.infoText}>Difficulty</Text>
+                <View style={styles.filterSectionImages}>
+                  <Image
+                    style={styles.eventTextFields}
+                    source={require('../../assets/images/icFilterDifficulty.png')}
+                  />
+                  <Text style={styles.tourDiffucultyTitle}>
+                    {tour.difficulty}
+                  </Text>
+                </View>
+              </View>
+
+              <View style={styles.languageSection}>
+                <Text style={styles.infoText}>Duration</Text>
+                <View style={styles.languageInputField}>
+                  <Image
+                    style={styles.eventTextFields}
+                    source={require('../../assets/images/icFilterDuration.png')}
+                  />
+                  <Text style={styles.languageTitle}>
+                    {tour.duration} hours
+                  </Text>
+                </View>
+              </View>
+            </View>
+            <View style={styles.eventDetails}>
+              <View style={styles.difficultyFilter}>
+                <Text style={styles.infoText}>Distance</Text>
+                <View style={styles.filterSectionImages}>
+                  <Image
+                    style={styles.eventTextFields}
+                    source={require('../../assets/images/icFilterDistance.png')}
+                  />
+                  <Text style={styles.tourDiffucultyTitle}>
+                    {tour.distance}
+                  </Text>
+                </View>
+              </View>
+
+              <View style={styles.languageSection}>
+                <Text style={styles.infoText}>Pet Friendly</Text>
+                <View style={styles.languageInputField}>
+                  <Image
+                    style={styles.eventTextFields}
+                    source={require('../../assets/images/icFilterPetFriendly.png')}
+                  />
+                  <Text style={styles.languageTitle}>
+                    {tour.petFriendly ? 'Yes' : 'No'}
+                  </Text>
+                </View>
+              </View>
+            </View>
+            <Text style={styles.infoText}>Audio</Text>
+            <View style={styles.inputFieldStyles}>
+              <Image
+                style={styles.eventTextFields}
+                source={require('../../assets/images/icFilterAudio.png')}
+              />
+              <Text style={{fontSize: 12}}>
```

Comment:
Move your inline style to the styles.js file

Reply:
Moved